api/routes/batch/delete.go

```diff
@@ -0,0 +1,31 @@
+package batch
+
+import (
+	"context"
+	"github.com/arkhn/fhir-river/api/topics"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/go-redis/redis"
+	"time"
+)
+
+// Delete deletes a batch by removing its Redis keys and Kafka topics
+func Delete(batchID string, rdb *redis.Client, admin *kafka.AdminClient) error {
+	if _, err := rdb.Del("batch:"+batchID+":counter").Result(); err != nil {
+		return err
+	}
+	if _, err := rdb.Del("batch:"+batchID+":resources").Result(); err != nil {
+		return err
+	}
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	batchTopics := []string{
+		topics.BatchPrefix + batchID,
+		topics.ExtractPrefix + batchID,
+		topics.TransformPrefix + batchID,
+		topics.LoadPrefix + batchID,
+	}
+	if _, err := admin.DeleteTopics(ctx, batchTopics, kafka.SetAdminOperationTimeout(60 * time.Second)); err != nil {
```

Comment:
Could `60 * time.Second` be an env var or something declared elsewhere?

Reply:
I don't know if the timeout duration is that important in this case even though we should log an event if this fails.

Reply:
agreed, not sure it deserves an env var. But we definitely should have visibility over this if it fails.